keel-core/src/main/kotlin/com/netflix/spinnaker/keel/enforcers/EnvironmentExclusionEnforcer.kt

```diff
@@ -109,12 +114,13 @@ class EnvironmentExclusionEnforcer(
   }
 
   /**
-   * @throws EnvironmentCurrentlyBeingActedOn if there's an active deployment
+   * @throws ActiveDeployments  if there's an active deployment
    */
   private fun ensureNoActiveDeployments(deliveryConfig: DeliveryConfig, environment: Environment) {
-    /**
-     * To be implemented in a future PR.
-     */
```

Comment:
why are you calling `artifactRepository.isDeployingTo` twice instead of just saving the result?

Reply:
Ugh, it's a typo (thinko?).  Thanks for spotting that!